v3/doio/kb12/kb12-01.json

```diff
@@ -0,0 +1,111 @@
+{
+    "name": "DOIO",
+    "vendorId": "0xD010",
+    "productId": "0x1201",
+    "keycodes": ["qmk_lighting"],
+      "menus": [
+      {
+        "label": "Lighting",
+        "content": [
+          {
+            "label": "Backlight",
+            "content": [
+              {
+                "label": "Brightness",
+                "type": "range",
+                "options": [0, 255],
+                "content": ["id_qmk_rgb_matrix_brightness", 3, 1]
+              },
+              {
+                "label": "Effect",
+                "type": "dropdown",
+                "content": ["id_qmk_rgb_matrix_effect", 3, 2],
+                "options": [
```

Comment:
![image](https://github.com/the-via/keyboards/assets/17491233/de828d76-db24-4f5a-bdff-cbf0d608cc15)

you are missing a lot of your RGB animations

Reply:
@Xelus22  Thank you for your suggestion. I've added it.

Reply:
@DOIO2022 since it looks liek you have all RGB effects enabled, you could replace all this block with just 
`"menus": ["qmk_rgb_matrix"],`